scripts/visualize/visualize_utils.R

```diff
@@ -251,30 +242,81 @@ visualize.map_thumbnail <- function(viz){
     # the way we did those was to have the class change only if the value changed, so it is compressed and number of 
     # classes likely != number of timesteps (i.e., if a never changed it would be p-1-1, if changed twice, p-1-1 p-32-2 p-34-1)
     precip.col.i <- as.numeric(sapply(precip.classes, FUN = function(x) {strsplit(x,'[-]')[[1]][3]}, USE.NAMES = FALSE))
-    t.diffs <- time.idx - precip.time.i
+    t.diffs <- time_idx - precip.time.i
     time.i <- which(t.diffs==min(t.diffs[t.diffs>=0]))
     counties@data$col[j] <- precip_cols[precip.col.i[time.i]]
   }
   
-  png(file = viz[['location']], height = viz$`fig-height`, width = viz$`fig-width`)
-  
-  
   par(mar=c(0,0,0,0), oma=c(0,0,0,0), bg = ocean_color, xaxs = 'i', yaxs = 'i')
-  sp::plot(counties, col = NA, expandBB = eval(parse(text = viz[["expandBB"]]))) #decimals not working?
-  sp::plot(states, col = state_color, add = TRUE)
-  sp::plot(islands, add = TRUE, col = island_color)
-  sp::plot(counties, add = TRUE, col = counties@data$col)
+  sp::plot(counties, col = NA, border = "#c6c6c6",
+           expandBB = eval(parse(text = expandBB))) #decimals not working?
+  sp::plot(states, col = state_color, border = "#c6c6c6", add = TRUE)
+  sp::plot(islands, add = TRUE, border = "#c6c6c6", col = island_color)
+  sp::plot(counties, add = TRUE, border = "#c6c6c6", col = counties@data$col)
   sp::plot(hurricane_track, add=TRUE, col = "black", lwd=3)
   cols <- rep("#FFFFFF00", length(storm))
-  cols[time.idx] <- 'red'
+  cols[time_idx] <- storm_color
```

Comment:
The never ending battle ... `.` or `_` !

Reply:
lol ... yeah, just trying to be consistent within the function